vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -104,6 +107,9 @@
     // tell if a confirmation dialog must be displayed to validate the user ids list
     private boolean mAddConfirmationDialog;
 
+    // The list of userId to invite for the room creation
```

Comment:
I would replace this comment with:
// The list of the identifiers of the current selected contacts 
// The type of these identifiers depends on the mContactsFilter:
// - matrix id when mContactsFilter = ContactsFilter.TCHAP_ONLY
// - email address when mContactsFilter = ContactsFilter.NO_TCHAP_ONLY
// - both in the other cases